website/source/docs/commands/acl/acl-role.html.md.erb

```diff
@@ -0,0 +1,315 @@
+Service Identities:
+   archiver (Datacenters: dc2)
+```
+
+## `read`
+
+Command: `consul acl role read`
+
+This command reads and displays a roles details.
+
+### Usage
+
+Usage: `consul acl role read [options] [args]`
+
+#### Options
+
+* [Common Subcommand Options](#common-subcommand-options)
+
+* `-id=<string>` - The ID of the role to read. It may be specified as a unique ID
+   prefix but will error if the prefix matches multiple role IDs.
+
+* `-meta` - Indicates that role metadata such as the content hash and raft
+  indices should be shown for each entry.
+
+* `-name=<string>` - The name of the role to read.
+
+### Examples
+
+Get role details:
+
+```sh
+$ consul acl role read -id 57147d87-6bf7-f794-1a6e-7d038c4e4ae9
+ID:           57147d87-6bf7-f794-1a6e-7d038c4e4ae9
+Name:         crawler
+Description:  web crawler role
+Policies:
+   2f8f99c7-edd9-2f09-7e4b-a1f519eb4fc2 - crawler-kv
+```
+
+Get role details by name:
+
+```sh
+$ consul acl role read -name archiver
+ID:           a365fdc9-ac71-e754-0645-7ab6bd747301
+Name:         archiver
+Description:  archiver role
+Service Identities:
+   archiver (Datacenters: dc2)
+```
+
+## `update`
+
+Command: `consul acl role update`
+
+This command is used to update a role. The default operations is to merge the
+current role with those values provided to the command invocation. Therefore to
+update just one field, only the `-id` or `-name` options and the option to
+modify must be provided. Note that renaming roles requires both the `-id` and
+`-name` as the new name cannot yet be used to lookup the role.
+
+### Usage
+
+Usage: `consul acl role update [options] [args]`
+
+#### Options
+
+* [Common Subcommand Options](#common-subcommand-options)
+
+* `-description=<string>` - A description of the role.
+
+* `-id=<string>` - The ID of the role to update. It may be specified as a
+  unique ID prefix but will error if the prefix matches multiple role IDs
+
+* `-meta` - Indicates that role metadata such as the content hash and raft
+  indices should be shown for each entry
+
+* `-name=<string>` - The role name.
+
+* `-no-merge` - Do not merge the current role information with what is provided
+  to the command. Instead overwrite all fields with the exception of the role
+  ID which is immutable.
+
+* `-policy-id=<value>` - ID of a policy to use for this role. May be specified
+  multiple times
+
+* `-policy-name=<value>` - Name of a policy to use for this role. May be
+  specified multiple times
+
+* `-service-identity=<value>` - Name of a service identity to use for this
+  role. May be specified multiple times. Format is the `SERVICENAME` or
+  `SERVICENAME:DATACENTER1,DATACENTER2,...`
+
+### Examples
+
+Update a role:
+
+```sh
+$ consul acl role update -id 57147d87-6bf7-f794-1a6e-7d038c4e4ae9 \
+    -description 'better description' -service-identity 'crawler'
+Role updated successfully
+ID:           57147d87-6bf7-f794-1a6e-7d038c4e4ae9
+Name:         crawler
+Description:  better description
```

Comment:
kait: can we add a real description forthese or just put "A description of the role"